### DISCUSSION TITLE
Table: Split delimited text into multiple cells/rows on paste [Feature Request]

### DISCUSSION THREAD

**Discussion by KeithGillette** [primefaces/community#781] [PrimeNG]
**Current behavior**
Pasting (tab/return) delimited text into an editable cell in the `p-table` component pastes the entire string, including delimiters, into the single cell input.

**Expected behavior**
Pasting delimited text into a cell in the Table component would paste the text into multiple cells in an editable row, split on the delimiters, and preferably add rows on newline/return.

**What is the motivation / use case for changing the behavior?**
* Data entry would be greatly speeded with the ability to paste into multiple editable cells.
* UX would match behavior of similar grids such as spreadsheets and word processor tables.

(Transferred from [Issue #8779](https://github.com/primefaces/primeng/issues/8779).)

**Comment by cetincakiroglu** [comment]
Hi,

To implement new feature requests, community engagement is required. We will reassess your request once the engagement increases.
